csrc/extensions.h

```diff
@@ -0,0 +1,9 @@
+
+#include <torch/torch.h>
```

Comment:
Why do we now include `torch.h` rather than `extension.h`?

Reply:
`extension.h` includes `Python.h` so at least in the non-python builds we can't have it.

https://github.com/pytorch/pytorch/blob/e88c999da38f30da123e14bdf0898bc475c8d04a/torch/extension.h#L1-L6

However the library doesn't seem to use any `<torch/python.h>` features as operators seem to be exported via [`RegisterOperators`](https://github.com/pytorch/pytorch/blob/bd6ec4efb49c3cfe06d728961bae0ecbe4469009/torch/csrc/jit/runtime/custom_operator.h#L16-L29).

We could still keep including `extensions.h` in the Python builds and only remove them in the non-python builds if you think it's safer.